### PR TITLE
Team page is now finished. Routes are setup with a way back to home page.

### PR DESCRIPTION
**Pull request by sohearn1**
I would like to go back later and pass the data for the team members using a json file instead of doing it inline with an array variable. I had to do some hacky stuff to get the flex layout to chop up the row size for the last two rows, but hey it works. Looks fine on mobile minus a little offset on the last card due to the above hack. I can fix that soon.

### CODE REVIEW

File: src/assets/data/team/team-members.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
+[
```
Comment (Kyle1668) [inline src/assets/data/team/team-members.json:1]: We need to add everyone's LinkedIn information

File: src/assets/data/team/team-members.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  { "name": "Jennifer Dutra", "title": "President", "imgRef": "jen-dutra.jpg" },
+  {
+    "name": "Daniel Segobiano",
+    "title": "President",
+    "imgRef": "daniel-segobiano.jpg"
+  },
+  {
+    "name": "Kyle O'Brien",
+    "title": "Lead Engineer",
```
Comment (Kyle1668) [inline src/assets/data/team/team-members.json:16]: Change title to "Engineering Director"

File: src/scss/components/team/_team.scss [hunk 3]
```diff
@@ -0,0 +1,86 @@
+.team {
```
Comment (Kyle1668) [inline src/scss/components/team/_team.scss:1]: On my screen, the card appears wider than what I see in the high fidelity mocks. Can you experiment with narrower cards? 

image

**Review by Kyle1668 — CHANGES_REQUESTED** [review]

**Review by ejmockler — CHANGES_REQUESTED** [review]
Great start Sean!

The team page needs a few more components— I'm guessing the navbar is still WIP? I don't think clicking the lightbulb to go back to the homepage is intuitive enough. It might help if you copy the lightbulb animation CSS from last year's website. But we need a navbar! In the navbar for the team page, have a link to home (if not already on the homepage), and likewise, a link to the team page on the home page. It'd be great if you could automatically hide the navbar link for the current page. 

We'll also need to fill in the Linkedin links before deployment, we can split this task if it's too much profile-hunting.

Also need to include the footer component on this page, it's located inside Joseph's background component for the landing but feel free to change the folder structure. 

One last thing, it'd be nice to reflect the "lead engineer" positions accurately. The leaders here are all "directors", and Kyle & I split this tedious directorship. It was more of a last-minute decision literally at our first group meeting, but we're both "Engineer Co-Director"! It's just not reflected in the organizer family doc Design used for reference: https://docs.google.com/document/d/17LFMK8Tk1HvsZ8Y7Ro_SC0RnayLwwMko2BuBb8aqCZA

Thanks for getting this progress up!

File: src/assets/data/team/team-members.json (hunk 1 above)
Comment (sohearn1) [reply src/assets/data/team/team-members.json:1]: Yeah, is there somewhere I can get all that?

File: src/assets/data/team/team-members.json (hunk 2 above)
Comment (sohearn1) [reply src/assets/data/team/team-members.json:16]: You got it.

Comment (sohearn1) [reply src/assets/data/team/team-members.json:16]: Also in reference to the wide cards. Can I get your screen resolution as a reference for a good baseline? I have a good idea of what values to use now.

Comment (ejmockler) [reply src/assets/data/team/team-members.json:16]: to keep the cards from overextending, just add a max-width. don't overcomplicate with too many media breakpoints!

Comment (Kyle1668) [reply src/assets/data/team/team-members.json:16]: Try Eric's suggestion

**Pull request merged**